README.md

```diff
@@ -104,7 +104,7 @@ Before setting up remote authentication, you will need to generate the `rpcauth`
 Example:
 
 ```sh
-❯ curl -sSL https://raw.githubusercontent.com/litecoin-project/litecoin/master/share/rpcuser/rpcuser.py | python - <username>
+❯ curl -sSL https://raw.githubusercontent.com/litecoin-project/litecoin/master/share/rpcauth/rpcauth.py | python - <username>
```

Comment:
@frranck please fix also the text above at ```.. the official `rpcuser.py` script ...```.

Reply:
@joaopaulofonseca not sure what you mean

Reply:
Just pointed out that, since we're fixing the example to mention the new script `rpcauth.py`, we should do the same [here](https://github.com/uphold/docker-litecoin-core/pull/15/files#diff-04c6e90faac2675aa89e2176d2eec7d8R102), with something like this ```... or use the official `rpcauth.py` script. ```